src/program.cpp

```diff
@@ -52,6 +52,32 @@ struct membuf : public std::streambuf {
         auto send = reinterpret_cast<char*>(end);
         setp(sbegin, send);
     }
+    virtual pos_type seekoff(off_type off, std::ios_base::seekdir dir,
+                             std::ios_base::openmode which) {
+        if (off != 0)
+            return -1;
```

Comment:
Not sure I understand, the implementation only supports an offset of 0? Looking at the code below, you might have intended the opposite. Also, why treat 0 as an error? Am I missing something?

Reply:
Oops, that was some code I had for local debugging, removed.